extensions-core/kubernetes-extensions/pom.xml

```diff
@@ -35,9 +35,20 @@
   </parent>
 
   <properties>
-    <kubernetes.client.version>11.0.4</kubernetes.client.version>
+    <kubernetes.client.version>19.0.0</kubernetes.client.version>
   </properties>
 
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+          <groupId>com.squareup.okhttp3</groupId>
+          <artifactId>okhttp</artifactId>
+          <version>4.12.0</version>
```

Comment:
any reason this is defined in dependencyManagement? it doesn't look like we define this as a dependency anywhere in this extension

Reply:
may want to add a comment to explain what this is helping override.

Reply:
This is to update okio version brought in by kubernetes-client

Reply:
will do. I have an impression that it might be remnant from another fix attempt.